src/scripts/changelog.zig

```diff
@@ -57,7 +57,8 @@ fn format_changelog(buffer: std.ArrayList(u8).Writer, options: struct {
     , .{options.today});
 
     var merges_left = options.merges;
-    for (0..128) |_| {
+    // TODO Shrink this down after we release again.
+    for (0..256) |_| {
```

Comment:
We have surpassed a backlog of 128 PRs, so I had to increase this (temporarily!).